Common/AR/Ar.md

```diff
@@ -0,0 +1,197 @@
+and synchronization with the Runtime SDK's `SceneView`. The `ArcGISArView` is responsible for starting and
+managing an `ARKit` or `ARCore` session. It uses a `LocationDataSource` for getting an initial GPS location
+and when continuous GPS tracking is required.
+
+### Features of the AR component
+
+- Allows display of the live camera feed
+- Manages `ARKit` or `ARCore` session lifecycle
+- Tracks user location and device orientation through a combination of `ARKit`or `ARCore` and the device GPS
+- Provides access to an `SceneView` to display your GIS 3D data over the live camera feed
+- `screenToLocation` method to convert a screen point to a real-world coordinate
+- Easy access to all `ARKit` or `ARCore` and `LocationDataSource` delegate methods
+
+## ARKit installation
+
+For the details to uses `ARKit`, pleases refere to the [Apple's documentation](https://developer.apple.com/augmented-reality).
+
+To create an application with the `ARKit` support:
+
+1. Downloads the sources of the [ArcGIS Runtime API Toolkit](https://github.com/Esri/arcgis-runtime-toolkit-qt).
+Sets the `AR_TOOLKIT_SOURCE_PATH` variable to the path of the sources of the toolkit.
+
+2. Installs the ArcGIS Runtime SDK for Qt version 100.6
+
+3. In Qt Creator, creates a new project "ArcGIS Runtime 100.6 Qt Quick C++ app" or "ArcGIS Runtime 100.6 Qt Quick QML app".
+
+4. In the projet file (.pro), includes the .pri file corresponding to the API used for the project:
+
+```
+include($$AR_TOOLKIT_SOURCE_PATH/Plugin/CppApi/ArCppApi.pri) // for C++ API
+```
+or
+```
+include($$AR_TOOLKIT_SOURCE_PATH/Plugin/QmlApi/ArQmlApi.pri) // for QML API
+```
+
+5. Updates the `Info.plist` file to request persmission for camera:
+
+```
+<key>NSCameraUsageDescription</key>
+<string>Camera access is needed for AR testing</string>
+```
+
+See [NSCameraUsageDescription Apple's documentation](https://developer.apple.com/documentation/bundleresources/information_property_list/nscamerausagedescription?language=objc) for details.
+
+6. The project is ready to be build and run in an iOS device. You need to verify the compatibility of your device
+with the ARKit.
+
+## ARCore installation
+
+For the details to uses `ARCore`, pleases refere to the [Google's documentation](https://developers.google.com/ar/).
+The details of the installation of the `ARCore` is describes in the
+[documentation](https://developers.google.com/ar/develop/c/enable-arcore).
+
+To create an application with the `ARCore` support:
+
+1. Downloads the sources of the [ArcGIS Runtime API Toolkit: https://github.com/Esri/arcgis-runtime-toolkit-qt.
+The `AR_TOOLKIT_SOURCE_PATH` variable refers to the path of the sources of the toolkit.
+
+2. Downloads and extracts the ARCore SDK for Android v1.10: https://github.com/google-ar/arcore-android-sdk/releases.
+This project contains the C header file (`arcore_c_api.h`) used to build the ARCore wrapper.
```

Comment:
So we need to download the SDK from the above link so that we can access `arcore_c_api.h`? This PR contains this header in `3rdparty` already. Is this step still required?